modules/common/src/Storage/JobStore.php

```diff
@@ -26,14 +27,21 @@ class JobStore extends AbstractDatabaseTable {
    *   Table name for this jobstore table.
    * @param \Drupal\Core\Database\Connection $connection
    *   Database connection.
+   * @param \Symfony\Component\EventDispatcher\EventDispatcherInterface $eventDispatcher
+   *   The event dispatcher service.
    * @param string $deprecatedTableName
    *   (Optional) Deprecated table name, if there is one.
    */
-  public function __construct(string $tableName, Connection $connection, string $deprecatedTableName = '') {
+  public function __construct(
+    string $tableName,
+    Connection $connection,
+    EventDispatcherInterface $eventDispatcher,
+    string $deprecatedTableName = ''
+  ) {
```

Comment:
Wondering why you added the eventDispatcher before $deprecatedTableName? Usually when you add new arguments to any function signature, especially a class constructor, you want to add them to the end to maintain compatibility with existing code or at least produce more readable errors.